frontend/dockerfile/docs/reference.md

```diff
@@ -804,18 +802,16 @@ $ docker buildx build --ssh default=$SSH_AUTH_SOCK .
 You can also specify a path to `*.pem` file on the host directly instead of `$SSH_AUTH_SOCK`.
 However, pem files with passphrases are not supported.
 
-## RUN --network
+### RUN --network
 
-> **Note**
->
-> Added in [`docker/dockerfile:1.1`](#syntax)
+```dockerfile
```

Comment:
Why the removal of the minimum versions for the older flags?

Reply:
References to very old versions (e.g. 1.1) were kinda noisy I thought, and at some point we'd probably want a cleanup, for example I think we might still carry references to Docker 17.x in some places. But maybe n-2 from stable is too aggressive.

With your other suggestion of adding it in the list it's less intrusive, so let's try that

Reply:
cc @crazy-max wdyt about version callout for these flags? See also Tonis' other comment https://github.com/moby/buildkit/pull/4688/files#r1503506056

I wonder if we should have a table somewhere summarizing the new syntax features and when they were introduced (or what their status is, if they're still in labs for example)